variables.tf

```diff
@@ -15,7 +15,7 @@ variable "prefix" {
 }
 
 variable "vpc_name" {
-  description = "Name of the VPC where the Endpoint Gateways will be created. This value is used to dynamically generate VPE names."
+  description = "Name of the VPC where the Endpoint Gateways will be created. This value is used to generate VPE names dynamically and must be specified before applying the changes."
```

Comment:
I don't think this description is going to make it anymore clear what the issue really is. The primary issue is that this variable:
1. Is used ONLY to generate VPE names (so that first part that says "Name of the VPC where" is misleading)
2. Whatever string is provided here MUST be a value that is known to terraform before ANY deployment has occurred, such as a static string or an input variable from the user

Reply:
Hi @toddgiguere,
Thank you for you comments. Just need to make sure of one thing that apart from the change in variable description, do you suggest to make some change in the logic to first verify whether the VPC provided is correct/already exists otherwise error out?
```hcl
data ibm_is_vpc get_vpc {
    name = var.vpc_name
}
```
Getting below output for incorrect vpc/ vpc that does not exists :

![image](https://github.com/user-attachments/assets/660c0183-f577-4367-97e1-e9c19d34c1f7)

Reply:
@imprateeksh No we do not need to validate the value of that name, it is only used as a prefix to create the VPE name and can actually be any value.

I don't see that code block above, `data.ibm_is_vpc.get_vpc`, anywhere in this module or examples?

Reply:
Added @SirSpidey as reviewer, he may be able to help with the wording.

Here is initial issue: https://github.com/terraform-ibm-modules/terraform-ibm-vpe-gateway/issues/559

Reply:
Sure thank you @toddgiguere , this clarifies. I used `data.ibm_is_vpc.get_vpc` in a separate example to ask you whether this is what we need to include. Now this is clear.